calpip/Task1/Task 1-2.txt

```diff
@@ -1,24 +1,24 @@
 Dear Sir or Madam,
 
 A couple of months ago I bought a long term membership card to one of 
-your gyms located in Winnipeg downtown area. I have paid a considerable 
-high amount of money expecting the best quality of service in return. 
+your gyms located in Winnipeg downtown area. I have paid a considerably 
+high amount of money, expecting the best quality of service in return. 
 Sadly, my experience has been really bad so far.
 
-For one thing, the main reason why I have become a customer was the 
-variety of fitness class announced in your flyers. However, most of 
+For one, the main reason why I became a customer was for the 
+variety of fitness classes announced in your flyers. However, most of 
```

Comment:
`For one` is correct when speaking, but when written it looks incomplete. Use `For one thing`.

`why I have become` can be  `why I became`, just a suggestion

`became a customer was the variety` -> `became a customer was for the variety`
You can also put `because` where `for` is.